packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -570,6 +571,9 @@ export function useClaimCctp(tx: MergedTransaction) {
         throw new Error('Transaction failed')
       }
     } catch (e) {
+      if (isUserRejectedError(e)) {
```

Comment:
Wouldn't `Sentry.captureException(e)` take care user-rejected-error now that the check is present inside the main function?

Reply:
Good point. I need to check first if re throwing error (on L577) for user rejection is fine

Reply:
Removed, it's called in `TransactionsTableRowAction` and we check for userRejected error